consensus-engine/src/utils.rs

```diff
@@ -0,0 +1,66 @@
+#[cfg(feature = "serde")]
+pub(crate) mod serde {
```

Comment:
this should probably go in some more general module instead of consensus engine

Reply:
Yeah, cleaner, move it to the `nomos-utils` crate.